firestore/cloud-client/snippets.py

```diff
@@ -815,3 +816,63 @@ def delete_collection(coll_ref, batch_size):
     # [END delete_full_collection]
 
     delete_collection(db.collection(u'cities'), 10)
+
+
+@pytest.mark.skip(reason="Dependant on a composite index being created,"
+                         "however creation of the index is dependent on"
+                         "having the admin client and definition")
+def collection_group_query():
+    db = firestore.Client()
+
+    # [START fs_collection_group_query_data_setup]
+    cities = db.collection(u'cities')
+
+    cities.document(u'SF').collection(u'landmarks').document().set({
+        u'name': u'Golden Gate Bridge',
+        u'type': u'bridge'
+    })
+    cities.document(u'SF').collection(u'landmarks').document().set({
+        u'name': u'Legion of Honor',
+        u'type': u'museum'
+    })
+    cities.document(u'LA').collection(u'landmarks').document().set({
+        u'name': u'Griffith Park',
+        u'type': u'park'
+    })
+    cities.document(u'LA').collection(u'landmarks').document().set({
+        u'name': u'The Getty',
+        u'type': u'museum'
+    })
+    cities.document(u'DC').collection(u'landmarks').document().set({
+        u'name': u'Lincoln Memorial',
+        u'type': u'memorial'
+    })
+    cities.document(u'DC').collection(u'landmarks').document().set({
+        u'name': u'National Air and Space Museum',
+        u'type': u'museum'
+    })
+    cities.document(u'TOK').collection(u'landmarks').document().set({
+        u'name': u'Ueno Park',
+        u'type': u'park'
+    })
+    cities.document(u'DC').collection(u'landmarks').document().set({
+        u'name': u'National Museum of Nature and Science',
+        u'type': u'museum'
+    })
+    cities.document(u'BJ').collection(u'landmarks').document().set({
+        u'name': u'Jingshan Park',
+        u'type': u'park'
+    })
+    cities.document(u'BJ').collection(u'landmarks').document().set({
```

Comment:
Could you break apart the document ref? It's longer than I'd expect it to be.

```
bj_collection = cities.document(u'BJ').collection(u'landmarks')
bj_collection.document().set( {
  u'name': u'Beijing Ancient Observatory',
  u'type': u'museum'
}
```

Reply:
done.